src/io/io_bosstiary.cpp

```diff
@@ -196,7 +196,7 @@ void IOBosstiary::addBosstiaryKill(std::shared_ptr<Player> player, const std::sh
 
 	int32_t value = player->getStorageValue(STORAGEVALUE_PODIUM);
 	if (value != 1 && newBossLevel == 2) {
-		auto returnValue = g_game().addItemStoreInbox(player, ITEM_PODIUM_OF_VIGOUR);
+		auto returnValue = g_game().processHouseOffer(player, ITEM_PODIUM_OF_VIGOUR);
```

Comment:
This item is earned, not purchased. Isn't it confusing that the function change says it was purchased from the store?